packages/webhooks/lib/utils.ts

```diff
@@ -76,13 +76,11 @@ export const filterHeaders = (headers: Record<string, string>): Record<string, s
 export const shouldSend = ({
     webhookSettings,
     success,
-    type,
-    operation
+    type
 }: {
     webhookSettings: DBExternalWebhook;
     success: boolean;
-    type: 'auth' | 'sync' | 'forward';
-    operation: SyncOperationType | AuthOperationType | 'incoming_webhook';
```

Comment:
we were only using operation for auth webhook, meaning we were passing some fake operation for the rest.
I removed `operation` and splitted `auth` type into `auth_creation` and `auth_refresh`